TruePath/AbsoluteDirectory.cs

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2024 TruePath contributors <https://github.com/ForNeVeR/TruePath>
+//
+// SPDX-License-Identifier: MIT
+
+namespace TruePath;
+
+public static class AbsoluteDirectory
+{
+    public static void CreateDirectory(AbsolutePath path) => Directory.CreateDirectory(path.Value);
+    public static void Delete(AbsolutePath path) => Directory.Delete(path.Value);
+    public static void Delete(AbsolutePath path, bool recursive) => Directory.Delete(path.Value, recursive);
```

Comment:
Redesign proposal:
```csharp
    public static void Delete(this AbsolutePath path, bool recursive = false);
    public static void RecursivelyDelete(this AbsolutePath path) => path.Delete(path.Value, recursive: true);
```

This should work for both files and directories.

Reply:
I would go with
```
public static void DeleteDirectory(AbsolutePath path) => Directory.Delete(path.Value);
public static void DeleteDirectory(AbsolutePath path, bool recursive) => Directory.Delete(path.Value, recursive);
```
since I cannot combine this without check if it's file or directory.

Reply:
Alright, let's have only one method then.
```
public static void DeleteDirectory(AbsolutePath path, bool recursive = false)
```